lib/msal-angular/docs/v2-docs/configuration.md

```diff
@@ -145,3 +146,108 @@ export function MSALGuardConfigFactory(): MsalGuardConfiguration {
 })
 export class AppModule { }
 ```
+
+## platformBrowserDynamic
+
+If you need to dynamically configure MSAL Angular (e.g. based on values returned from an API), you can use `platformBrowserDynamic`.
```

Comment:
Can you add a little explanation about what platformBrowserDynamic is and/or a link that explains it?

Reply:
^^

Reply:
Done!